tenets/codelingo/psr2/lower-case-constant-values/example-orig.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+function helloWorld() {
+    print "hello world!";
+    $thing = true;
```

Comment:
Should these examples be bad? I get no issues found with the example source

Reply:
Fixed